app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -260,10 +260,11 @@ def placement_groups
   def sshkeys
     require "sshkey"
     collector.sshkeys.each do |tkey|
+      public_key = tkey.ssh_key.tr("\n", ' ')
       persister.auth_key_pairs.build(
         :name        => tkey.name,
-        :public_key  => tkey.ssh_key,
-        :fingerprint => SSHKey.sha1_fingerprint(tkey.ssh_key)
+        :public_key  => public_key,
+        :fingerprint => SSHKey.sha1_fingerprint(public_key)
```

Comment:
@agrare Unrelated but weren't we just talking about this gem?

Reply:
Actually no this is a different gem (https://github.com/heyvito/ssh-fingerprint)